libraries/drivers_db_sqlite.rb

```diff
@@ -16,6 +31,32 @@ def out
       def url(deploy_dir)
         "sqlite://#{deploy_dir}/shared/#{out[:database]}"
       end
+
+      private
+
+      # rubocop:disable Metrics/AbcSize, Metrics/MethodLength
+      def link_sqlite_database
+        deploy_to = deploy_dir(app)
+        relative_db_path = out[:database]
+        shared_directory_path = File.join(deploy_to, 'shared', relative_db_path.sub(%r{/[^/]+\.sqlite3?$}, ''))
+        release_path = Dir[File.join(deploy_to, 'releases', '*')].last
+
+        context.directory shared_directory_path do
+          recursive true
+          not_if { ::File.exist?(shared_directory_path) }
+        end
+
+        context.file File.join(deploy_to, 'shared', relative_db_path) do
+          action :create
+          not_if { ::File.exist?(File.join(deploy_to, 'shared', relative_db_path)) }
+        end
+
+        context.link File.join(release_path, relative_db_path) do
+          to db_path
```

Comment:
`db_path` is missing and tests are failing:
```
 [2017-10-20T05:03:35+00:00] ERROR: deploy[dummy_project] (opsworks_ruby::deploy line 22) had an error: NoMethodError: undefined method `db_path' for Chef::Resource::Link
```

Reply:
Oh that's embarrassing. Sorry for pushing a broken PR :-/

Fix coming up.